DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -198,6 +196,15 @@
         "version" : "1.1.3"
       }
     },
+    {
+      "identity" : "xcodeeditor",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/appsquickly/XcodeEditor.git",
+      "state" : {
+        "branch" : "master",
```

Comment:
They do have some tags, the latest is 1.6.4, from 2015 though :) do you need master branch because there are otherwise unreleased changes?